pkg/apiserver/domain/service/addon.go

```diff
@@ -400,8 +400,8 @@ func (u *addonServiceImpl) EnableAddon(ctx context.Context, name string, args ap
 	if err != nil {
 		return err
 	}
-	for _, r := range registries {
-		err = pkgaddon.EnableAddon(ctx, name, args.Version, u.kubeClient, u.discoveryClient, u.apply, u.config, r, args.Args, u.addonRegistryCache)
+	for i, r := range registries {
```

Comment:
Do we support enabling the addon from a specific registry? I noticed that the CLI supports it.

Reply:
This pr does this thing.

Reply:
But, here doesn't support it. We could add a param named registry in EnableAddonRequest.

Reply:
Goted. Will add a parameter in apiserver.

Reply:
Added.